packages/suite-desktop-core/e2e/tests/spawn-bridge.test.ts

```diff
@@ -53,6 +54,7 @@ test.describe.serial('Bridge', { tag: ['@group=suite', '@desktopOnly'] }, () =>
         await suite.window.title();
         const onboardingPage = new OnboardingActions(
             suite.window,
+            new AnalyticsActions(suite.window),
```

Comment:
I wonder whethere we shouldnt rather initilized  AnalyticsActions in the OnboardingActions and dont have to import it here and pass it in fixtures.ts
:thinking:

Reply:
Maybe, but there are currently cases of only using AnalyticsPage and not Onboarding. I would keep it like this for now.